_extension/package.json

```diff
@@ -57,6 +57,15 @@
                         "tags": [
                             "experimental"
                         ]
+                    },
+                    "typescript.native-preview.goMemLimit": {
+                        "type": "string",
+                        "description": "Set GOMEMLIMIT for the language server (e.g., '2048MiB', '4GiB'). See https://pkg.go.dev/runtime#hdr-Environment_Variables for more information.",
+                        "pattern": "^[0-9]+[KMGT]iB$",
```

Comment:
God help us if the `T` ever gets used 😆

Reply:
This isn't the same regex as what I pasted; this won't support just B for bytes. You should be able to copy it exactly.

Reply:
I didn't mean a throw, I meant to output to the log with `.error()` and just pretend that the memlimit was never set.